templates/home.html

```diff
@@ -1,8 +1,13 @@
 {{ define "content" }}
 
 <h1 class="text-primary">HELLO GOLANG</h1>
-{{ if .data.authenticatedUser  }}
+{{ include "partials/flashes" }}
+
+{{ if .data.authenticatedUser }}
```

Comment:
Ideally, later, it would be great to have something like `currentUser` set in middleware as discussed before. So it would not be in `.data` ⭐

Reply:
This will be implemented in upload csv PR